assets/src/edit-story/utils/copyPaste.js

```diff
@@ -183,3 +202,62 @@ export function getPastedCoordinates(originX, originY) {
     y: PAGE_HEIGHT - y > allowedBorderDistance ? y : placementDiff,
   };
 }
+
+/**
+ * Creates a clipboard event, but won't crash the app if
+ * used in internet explorer.
+ *
+ * @param {string} type Type of clipboard event. One of ['copy', 'cut', 'paste'].
+ * @param {DataTransfer} clipboardData The clipboard data to add to the clipboard event.
+ * @return {ClipboardEvent|null} The ClipboardEvent
+ */
+export function createClipboardEvent(type = 'copy', clipboardData) {
+  let evt;
+
+  try {
+    evt = new ClipboardEvent(type, {
+      clipboardData: clipboardData || new DataTransfer(),
+    });
+  } catch (error) {
+    // eslint-disable-next-line no-console
+    console.error(
+      'Copy failed. ClipboardEvent not supported in Internet Explorer.',
+      error
+    );
+    return null;
+  }
+
+  return evt;
+}
+
+/**
+ * Processes copied/cut content for preparing elements to add to clipboard.
+ *
+ * @param {Object} page Page which all the elements belong to.
+ * @param {Array} elements Array of story elements.
+ * @param {Array} animations Array of story animations.
+ */
+export async function addDataToClipboardWithNavigator(
+  page,
+  elements,
+  animations
+) {
+  if (!elements.length) {
+    return;
+  }
+
+  const { htmlContent, serializedPayload } = serializeTextAndHTMLData(
+    elements,
+    animations
+  );
+
+  // do it live here bruh.
+  try {
+    await navigator.clipboard.writeText(
+      `<!-- ${serializedPayload} -->${htmlContent}`
+    );
```

Comment:
This is the magic - can programmatically add stuff to the clipboard using the [clipboard api](https://developer.mozilla.org/en-US/docs/Web/API/Clipboard_API). This is a relatively new api, with some more complicated functions not being available on every browser yet. 
<img width="604" alt="Screen Shot 2021-06-09 at 4 24 15 PM" src="https://user-images.githubusercontent.com/22185279/121437250-3688f500-c93f-11eb-94a2-e92d4d297a8b.png">